packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart

```diff
@@ -21,22 +21,24 @@ import '../../src/fakes.dart';
 import '../../src/test_flutter_command_runner.dart';
 
 const String flutterRoot = r'C:\flutter';
-const String buildFilePath = r'C:\windows\CMakeLists.txt';
-const String visualStudioPath = r'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community';
-const String _cmakePath = visualStudioPath + r'\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe';
+const String buildFilePath = r'windows\CMakeLists.txt';
+const String visualStudioPath =
+    r'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community';
+const String _cmakePath = visualStudioPath +
+    r'\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe';
 const String _defaultGenerator = 'Visual Studio 16 2019';
 
 final Platform windowsPlatform = FakePlatform(
   operatingSystem: 'windows',
   environment: <String, String>{
     'PROGRAMFILES(X86)':  r'C:\Program Files (x86)\',
-    'FLUTTER_ROOT': flutterRoot,
```

Comment:
Per our [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#in-defense-of-the-extra-work-that-hand-formatting-entails), we don't use dart format. please revert all of these formatting changes.

Reply:
Looks like automatically applied formatting by VS Code. Just went through and manually reset all of these indentations. I believe that should not be an issue going forward now that I disabled "format on save" under "user/dart" in VS Code

Reply:
Cool! This might be worth noting on the onboarding doc, as almost everyone's first PR is autoformatted :)